RockWeb/Blocks/Event/CalendarLava.ascx.cs

```diff
@@ -625,7 +625,7 @@ private bool SetFilterControls()
             // Set filter visibility
             bool showFilter = pnlCalendar.Visible || rcwCampus.Visible || rcwCategory.Visible || drpDateRange.Visible;
             pnlFilters.Visible = showFilter;
-            pnlList.CssClass = showFilter ? "col-md-9" : "col-md-12";
+            pnlList.CssClass = showFilter ? "col-md-9" : "col-md-12 col-xs-12";
```

Comment:
This is a change to a core block. There's a good chance any future changes to this block won't include this change. Do we need it?

Reply:
@richarddubay not a big deal if it gets wiped out. It was from a previous iteration of events and we're not using that block in the final version. But it definitely won't hurt for it to be in there in case we decide to use it.

Reply:
What does "we're not using that block in the final version" mean?